Library/Homebrew/test/test_integration_cmds.rb

```diff
@@ -797,4 +797,41 @@ def test_custom_command
         cmd(cmd, {"PATH" => "#{path}#{File::PATH_SEPARATOR}#{ENV["PATH"]}"})
     end
   end
+
+  def test_search
+    formula_file = CoreTap.new.formula_dir/"testball.rb"
+    formula_file.write <<-EOS.undent
+      class Testball < Formula
+        desc "Some test"
+        url "https://example.com/testball-0.1.tar.gz"
+      end
+    EOS
+
+    desc_cache = HOMEBREW_CACHE.join("desc_cache.json")
```

Comment:
I would prefer if we used `/` more consistently when dealing with `Pathname` objects. We already do that in all our formulae and it would be nice to move in the same direction in the package manager code (at least for code that is being added, ironing out style issues in old code is a totally different issue). But really just a style nit.